modules/xcm-bridge-hub-router/src/lib.rs

```diff
@@ -194,6 +194,8 @@ impl<T: Config<I>, I: 'static> ExporterFor for Pallet<T, I> {
 		let fee_factor = Self::delivery_fee_factor();
 		let fee = fee_factor.saturating_mul_int(fee_sum);
 
+		// TODO: how to handle `maybe_payment`? subsume with `(T::FeeAsset::get(), fee)`?
```

Comment:
I think we can subsume and use it instead of `Config;:BaseFee`.

Reply:
@svyatonik 
I tried to fix maybe_payment, please, check https://github.com/paritytech/parity-bridges-common/pull/2313/commits/623c9c0e5feae38e948907a6b5f2f1c0354a3fb7